sway-fmt/src/fmt.rs

```diff
@@ -641,4 +641,23 @@ fn one_liner() -> bool {
         let (_, formatted_code) = result.unwrap();
         assert_eq!(correct_sway_code, formatted_code);
     }
+
+    #[test]
+    // This test is here to check that the `use` statements are properly formatted
+    // Currently, there is a hard-to-diagnose bug which adds a leading `--` to the first
+    // `use` statement
+    fn test_use_statement() {
+        let expected_sway = r#"script;
+use std::chain::{panic,log_u8};
+use std::chain::assert;
+use std::address::{Address,Address};
```

Comment:
Should we import the same thing twice? Isn't that an error?

Reply:
According to https://github.com/FuelLabs/sway/issues/727, it should not compile.

Reply:
Is it an issue that the formatter didn't raise an error? According to https://fuellabs.github.io/sway/latest/getting-started/temporary_workarounds.html#formatter, we only format compiling code.